README.md

```diff
@@ -1,5 +1,25 @@
 # rbxlx-to-rojo
 Tool to convert existing Roblox games into Rojo projects by reading their `rbxlx` place files.
 
+# Using rbxlx-to-rojo
+## Setup
+Before you can use rbxlx-to-rojo, you need to have Rojo 0.5.0 Alpha 12 to use the tool.
+You also need a rbxlx place file that has a script. If there aren't any scripts in the rbxlx file, rbxlx-to-rojo will return an error.
+
+Download the latest release of rbxlx-to-rojo here: https://github.com/rojo-rbx/rbxlx-to-rojo/releases
+## Porting the game
+Let's assume we have a rbxlx file named "game", with a script in ServerScriptService.
+
+### Steps to port the game:
+1. Double-click on rbxlx-to-rojo on wherever you installed it.
+2. Select a place file.
+3. Select the path where you want to place it.
+
+If you followed the steps correctly, you should see something that looks like this:
+https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png
+![](https://cdn.discordapp.com/attachments/609230525604954135/610155628786221117/unknown.png)
```

Comment:
Can you save this image in a folder in the repo? So instead of this, it'd be like `![](assets/folder.png)`

Reply:
Fixed. Check the latest commit.